ebill_paynet/data/ir_cron.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Add a copyright in XML files? (not blocking)

Reply:
I don't see any point doing it, except if there is some specific code in there, which is not the case.